Cauldron/Assets/Scripts/PlantsFilter.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using CauldronCodebase;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class PlantsFilter : MonoBehaviour
```

Comment:
It's okey for now, but ideally we should generalize those filters, so that we will have a couple of scripts and a model for them to work on)
The goal is to be able to change the model in a couple of code lines and the filters would adapt.